app/models/grda_warehouse/tasks/push_clients_to_cas.rb

```diff
@@ -89,6 +93,9 @@ def sync!
               project_client.enrolled_in_rrh = client.enrolled_in_rrh(enrollments)
               project_client.enrolled_in_psh = client.enrolled_in_psh(enrollments)
               project_client.enrolled_in_ph = client.enrolled_in_ph(enrollments)
+              project_client.ongoing_es_enrollments = client.processed_service_history&.cohorts_ongoing_enrollments_es&.map { |e| e['project_name'] + ': ' + e['date']&.to_date.to_s }.presence
+              project_client.ongoing_so_enrollments = client.processed_service_history&.cohorts_ongoing_enrollments_so&.map { |e| e['project_name'] + ': ' + e['date']&.to_date.to_s }.presence
```

Comment:
Should these project names be made safe?  Maybe we should repeat the logic for these as well, something like:
```suggestion
              project_client.ongoing_es_enrollments = client.processed_service_history&.cohorts_ongoing_enrollments_es&.map { |e| safe_project_names[e['project_id']] + ': ' + e['date']&.to_date.to_s }.presence
              project_client.ongoing_so_enrollments = client.processed_service_history&.cohorts_ongoing_enrollments_so&.map { |e| safe_project_names[e['project_id']] + ': ' + e['date']&.to_date.to_s }.presence
```
Unless those are already "safe".

Reply:
These names are already safe (set in GrdaWarehouse::WarehouseClientsProcessed.last_seen_in_type(type)), but setting them here would be more clear and provide some more consistency. I'll run a few quick tests and as long as this comes through as expected, i'll include it in the upcoming commit.